src/shrub/v3/evg_project.py

```diff
@@ -42,10 +44,12 @@ class EvgModule(BaseModel):
     branch: str
     prefix: str
 
-    def get_repository_name(self) -> str:
+    def get_repository_name(self) -> Optional[str]:
```

Comment:
This seems like a breaking change, but people will probably start noticing it when they upgrade. Is there any additional step we need to communicate about this change?

Reply:
Actually it should have been `Optional[str]` before too. External library could also return `None` but our type annotation was missing that.
Not sure what to do in such case.

Reply:
If that's the case, I expect callers out there should be able to handle it. The risk seems relatively low to me.